docs/source/tools/visual.rst

```diff
@@ -0,0 +1,32 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
Can we put an example output in the docs? ideally one generated from the quickstart, so we can tell users how to reproduce it.

Reply:
Will add it very soon. In a different PR